climada/entity/impact_funcs/test/test_imp_fun_set.py

```diff
@@ -288,6 +288,55 @@ def test_remove_add_pass(self):
         self.assertEqual([1], imp_fun.get_ids("TC"))
 
 
+class TestEquality(unittest.TestCase):
+    """Test equality method for ImpactFuncSet"""
+
+    def setUp(self):
+        intensity = np.array([0, 20])
+        paa = np.array([0, 1])
+        mdd = np.array([0, 0.5])
+
+        fun_1 = ImpactFunc("TC", 3, intensity, mdd, paa)
+        fun_2 = ImpactFunc("TC", 3, intensity, mdd, paa)
```

Comment:
Maybe use deep copies here to absolutely make sure that equality is ensured if the *values* are the same, and not the *objects*?
```suggestion
        fun_2 = ImpactFunc("TC", 3, deepcopy(intensity), deepcopy(mdd), deepcopy(paa))
```